index.js

```diff
@@ -151,6 +151,8 @@ function init (ssb, config) {
     link: {
       create: scuttle.link.create
     },
-    findByFeedId: scuttle.link.findGroupByFeedId
+    findByFeedId: scuttle.link.findGroupByFeedId,
+    get: keystore.group.get,
+    list: keystore.group.list
```

Comment:
There's a gotcha here (I think). While we want the keystore methods to be `sync` for fast box / unbox, we have a guard around calling `keystore.group.get` because we have `init` guards which guarentee the keystore is ready before these are called.

Exposing these methods generally _could_ get you no results ... because the keystore could still be spinning up. Note in your tests you only run list _after_ you've created a group (so the keystore is known to be ready).

**solution** : I think for the external API these methods need to be async, and made "patient"